cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-// Required polyfills for IE9.
-if ( window.Modernizr && !window.Modernizr.classlist ) { require( '../polyfill/class-list' ); } // eslint-disable-line no-undef, global-require, no-inline-comments, max-len
+require( '../polyfill/class-list' );
```

Comment:
Yeah, I mistakenly removed this code. I'm working on PR to remove them as we discussed last week. Fixed in https://github.com/cfpb/cfgov-refresh/commit/f217937d4eec3537f28765b2f956d29b1609df50